src/pages/index.html

```diff
@@ -4,16 +4,380 @@
 
 {% block content %}
 
-    <section class="section">
-        <div class="container">
-            <h1>{{ welcome.title }}</h1>
-            <p>{{ welcome.description }}</p>
+<div class="landing ">
+
+  <div class="container  ">
+             <div class="row ">
+              <div class="mb-3 col-xl-7 col-lg-7 col-md-7">
+
+  <div class="modern">
+    <h2>
+      Modern POS that makes running your business easier
+  </h2>
+  <div class="d-flex justify-content-cente align-items-center">
+    
+  <i class="fa-solid fa-check"></i>
+  <p>    Manage cost savings & payments method</p>
+  </div>
+  <div class="d-flex justify-content-cente align-items-center ">
+    <i class="fa-solid fa-check"></i>
+    <p>   Manage cost savings & payments method</p>
+    
+  </div>
+  <div class="trial">
+    <form action="">
+      <input type="email" placeholder="Enter Your Email" />
+      <input type="submit" value="Start Free Trial" />
+    </form>
+    <div class="log">
+<p> Already have an account? <a href="$">Log In now</a> </p>
+
+    </div>
+  </div>
+        
+  </div>
+
+ 
+</div>
+
+  <div class="mb-3 col-xl-5 col-lg-5 col-md-5  position-relative ;
+  ">
+
+  <div class="pic">
+    <div class="shap">
+      <img src="../assets/images/shape (2).png" alt="">
+
+      
+
+              </div>
+              <div class="squr1 position-absolute">
+                <div class="d-flex">
+                  <img src="../assets/images/Icon.svg" alt="">
+                    <span class="nomber2">+3.50%</span>
+                    <i class="fa-solid fa-arrow-up"></i>
+                  </div>
+     <div class=" d-flex flex-column">
+      <span class="nomber">86,590</span>
+      <span class="total">Total Customer</span>
+              </div>
+            </div>
+
+            <div class="squr2 position-absolute ">
+              <div class="d-flex">
+                <img src="../assets/images/aaaaa.svg" alt="">
+                <span class="nomber2">+48.50%</span>
+                <i class="fa-solid fa-arrow-up"></i>
+              </div>
+   <div class=" d-flex flex-column">
+    <span class="nomber">$36,845.00</span>
+    <span class="total">Total Revenue</span>
+  </div>
+   </div>
+
+
+        
+    <img src="../assets/images/person.png" alt="">
+  </div>
+
+</div>
+
+ </div>
+</div>
+
+
+
+</div>
+
+ <div class="page">
+  <div class="container">
+    <div class="developed">
+      <div class="updated">
+        <img src="../assets/images/icon (11).svg" alt="">
+        <p>  Developed with the highest care. Always updated. </p>
+      </div>
+      <div class="updated">
+        <img src="../assets/images/icon (12).svg" alt="">
+        <p> Get free goodies from all over the world after purchase </p>
+      
+      </div>
+      <div class="updated">
+        <img src="../assets/images/icon (13).svg" alt="">
+        <p>  Save up to 10% of every purchase you made </p>
+      
+      
+      </div>
+          </div>
```

Comment:
![Capture](https://github.com/shaban011147/landing/assets/106445705/f482179a-2195-49c0-aa66-e9b80f823ed8)
1- the space between div(updated) not same in design
2-why not font family in same design
3- there divs (updated) not center 
4- the first div (updated) not same icon in design

Reply:
done